orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -130,6 +135,12 @@ class DependentPipelineStarter implements ApplicationContextAware {
       callable = AuthenticatedRequest.propagate({
         log.debug("Destination thread user: " + AuthenticatedRequest.getAuthenticationHeaders())
         pipeline = executionLauncher().start(PIPELINE, json)
+
+        String id = registry.createId("pipelines.triggered")
```

Comment:
This is actually `Id`, not a string. You'll probably get some unintended side effects by casting it as a string

Reply:
Oops, changing.